Makefile

```diff
@@ -829,6 +854,30 @@ uninstall-minio:
 	@echo "Removing minio PVCs from $(MINIO_NAMESPACE)"
 	- @oc delete pvc -n $(MINIO_NAMESPACE) -l app.kubernetes.io/name=$(MINIO_CHART) --timeout=30s ||:
 
+.PHONY: install-loki
+install-loki:
+	@echo "→ Checking if loki-stack already exists in namespace $(OBSERVABILITY_NAMESPACE)"
```

Comment:
Instead of installing into namespace based on OBSERVABILITY_NAMESPACE, can we have it installed on openshift-logging where is the default namespace for logging? The reason is that currently Korrel8r won't work with non-default namespace installation for loki.